### DISCUSSION TITLE
Scripting language?

### DISCUSSION THREAD

**Discussion by C-BJ** [inanevin/LinaEngine#66] [Ideas]
What scripting languages will linaengine support?
If I were to suggest, I would suggest [Wren](https://github.com/wren-lang/wren)
This is also the scripting language used by [LuxeEngine](https://luxeengine.com/)

**Comment by inanevin** [comment]
I don't plan to support any scripting languages just yet. It is easy to widen your scope thinking about new features such as SLs, however it requires careful consideration, design/structural changes to compilation/toolset/build pipelines, as well as bunch of other game framework & performance related quirks. 

For the time being, it's not on the roadmap, the main development language using Lina will be C++.
